### ISSUE TITLE
Adding a verb to do bootstrap resampling for confidence interval.

### ISSUE THREAD

**Issue by SamuelLarkin** [johnkerl/miller#1670]
Hi,
I would like to suggest a new feature.  I would like to be able to do bootstrap resampling to get a confidence interval.  You can find an example of how it works here: [scipy.stats.bootstrap()](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.bootstrap.html).

**Comment by johnkerl** [comment]
@SamuelLarkin have you seen https://miller.readthedocs.io/en/latest/reference-verbs/#bootstrap ?

**Comment by SamuelLarkin** [comment]
@johnkerl yes I did, but I failed to see how I could sample do confidence interval using `bootstrap`.  I would need to sample `1000 x len(data)`, group the data in `1000` batches of `len(data)`, perform mean on each batch then do `symetric percentile 5%`.  How to I add the batch id in order to group the data in `1000` batches of `len(data)`?

**Comment by johnkerl** [comment]
Resolved by #2189
